### ISSUE TITLE
How do I remove weeks 1-6 in story mode?

### ISSUE THREAD

**Issue by NicklGames** [ShadowMario/FNF-PsychEngine#394]
I tried removing them from weekList, removed their .json files. THEY STILL EXIST!!

**Comment by Nissan101** [comment]
Go to the weeks folder and remove the .json files for each week. It's should have to game do the rest and remove them

**Comment by NicklGames** [comment]
> Go to the weeks folder and remove the .json files for each week. It's should have the game do the rest and remove them
no, that doesnt work. i already got it, thanks to some YT tutorial. Instead, you should go to week editor you need to "create" each week, AND mark the "Hide from Story Menu" checkbox

**Comment by ShadowMario** [comment]
> > Go to the weeks folder and remove the .json files for each week. It's should have the game do the rest and remove them
> 
> no, that doesnt work. i already got it, thanks to some YT tutorial. Instead, you should go to week editor you need to "create" each week, AND mark the "Hide from Story Menu" checkbox

It works though, but hey, if you already got it then close the damn issue

**Comment by NicklGames** [comment]
> > > Go to the weeks folder and remove the .json files for each week. It's should have the game do the rest and remove them
> > 
> > 
> > no, that doesnt work. i already got it, thanks to some YT tutorial. Instead, you should go to week editor you need to "create" each week, AND mark the "Hide from Story Menu" checkbox
> 
> It works though, but hey, if you already got it then close the damn issue

s a d
f a c e
